### PR TITLE
outputFileFormat with multi/single options

### PR DESCRIPTION
**Pull request by mcelotti**
#87

### CODE REVIEW

**Comment by mcelotti** [comment]
I thought the same thing but after looking at the code I decided to avoid that mainly to reduce the "outputFileFormat" param complexity.
Anyway if you think it's a good idea then I can make the change.

**Comment by christian-bromann** [comment]
> Anyway if you think it's a good idea then I can make the change.

Yeah, let's add this to avoid confusion about the new version.

File: test/reporter.spec.js [hunk 1]
```diff
@@ -106,6 +106,26 @@ with new line
...
+
+        after(() => {
+            rimraf.sync(outputDir)
+        })
+
+        it('should have used expected file name format', () => {
+            [ xml1, xml2 ] = fs.readdirSync(outputDir)
+            xml2.should.be.equal('some-file-0-0.xml')
+            xml3.should.be.equal('some-file-0-1.xml')
+        })
```
Comment (christian-bromann) [inline test/reporter.spec.js]: I think we need to wrap each check in an individual describe. So one describe for the fn format and one for the single/multi format

**Comment by christian-bromann** [comment]
Released as `0.4.0`

**Pull request merged**